eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -231,8 +261,15 @@ jobs:
   - job: 'Test'
     condition: and(succeeded(), ne(variables['Skip.Test'], 'true'))
 
+    # HACK: We can't reference parameters to define other parameters, so I am
```

Comment:
Given this wasn't needed for the other languages we must be a little inconsistent with java. @benbp once we get the matrix work in we should try and make these all consistent across languages.

Reply:
The consistency between templates has devolved quite a bit I'm seeing.